### PR TITLE
Cleanup Onboarding components (wave 2/n)

### PR DESCRIPTION
**Pull request by SamChou19815**
### Summary 

This PR cleans up the OnboardingReview component by removing all the unused functions and fields.

### Test Plan 

This is a presentational component, so you can just see the onboarding review page and check everything is properly displayed.

Screen Shot 2021-01-30 at 16 18 26

### CODE REVIEW

**Review by tcho6319 — CHANGES_REQUESTED** [review]
Just commented a nit and noticed a discrepancy in the review page when comparing this PR and master. 

The staging site for this PR is on the left and the staging site for master is on the right in the screenshot. The total transfer credits is 0 credits for the left but it's 4 credits on the right. I don't think any credits are assigned to the AP Credits that I put for the left. The issue is the same for IB credits as well.
Screen Shot 2021-01-31 at 9 51 26 AM

File: src/components/Modals/Onboarding/OnboardingReview.vue [hunk 1]
```diff
@@ -390,15 +235,7 @@ export default Vue.extend({
       }
       const exam = {};
       for (const sect of sections) {
```
[the remark this answers is not in the document]
Comment (SamChou19815) [reply src/components/Modals/Onboarding/OnboardingReview.vue]: changed.

**Comment by SamChou19815** [comment]
> Just commented a nit and noticed a discrepancy in the review page when comparing this PR and master.
> 
> The staging site for this PR is on the left and the staging site for master is on the right in the screenshot. The total transfer credits is 0 credits for the left but it's 4 credits on the right. I don't think any credits are assigned to the AP Credits that I put for the left. The issue is the same for IB credits as well.
> Screen Shot 2021-01-31 at 9 51 26 AM

@tcho6319 fixed.

**Review by tcho6319 — APPROVED** [review]
LGTM!

**Pull request merged**